src/components/deactivate-account-confirmation-drawer/DeactivateAccountConfirmationDrawer.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback } from 'react'
+
+import {
+  getModalVisibility,
+  setVisibility
+} from 'audius-client/src/common/store/ui/modals/slice'
+import { deactivateAccount } from 'audius-client/src/containers/deactivate-account-page/store/slice'
+import { StyleSheet, View } from 'react-native'
+
+import Text from 'app/components/text'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+import ActionDrawer from '../action-drawer'
+
+const MODAL_NAME = 'DeactivateAccountConfirmation'
+const messages = {
+  confirmTitle: 'Deactivate Account',
+  confirm: 'Are you sure? This cannot be undone.',
+  buttonDeactivate: 'Deactivate',
+  buttonGoBack: 'Go Back'
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    customDrawerTitle: {
+      maxHeight: 500,
+      borderBottomColor: themeColors.neutralLight8,
+      borderBottomWidth: 1
+    },
+    customDrawerTitleHeader: {
+      marginTop: 8,
+      marginBottom: 16,
+      textAlign: 'center',
+      fontSize: 18,
+      fontWeight: 'bold'
```

Comment:
I'm pretty sure the `fontWeight` style doesn't have any effect. You can use the `weight` prop on the `Text` component though

Reply:
good callout - looks like it only works for iOS